event/templates/event/profile_template.html

```diff
@@ -42,7 +42,7 @@ <h1>{{event.title}}</h1>
                 <button id="join-event-btn-{{event.id}}" class="join-event-btn join" data-join-event-id="{{event.id}}">Join this event</button>
                 {% endif %}
             {% else %}
-                {% if user.is_authenticated and user != event.author %}
+                {% if user.is_authenticated and user != event.creator %}
```

Comment:
maybe those are the parts that have been making server error..?

Reply:
I don't think this would cause a server error but maybe?